config.js.template

```diff
@@ -40,6 +40,14 @@ module.exports = {
   },
   */
 
+  // Integration test configs
```

Comment:
I don't think we should add anything in this file. This is the template for `src/config.js` and you are adding some test template for `__tests__/integration/configuration/config.js.template`.

We shouldn't even have a template for that, I think.

Reply:
The reason for this template is that on the integration tests a full instance of the Wallet Headless has to be started, with the correct configuration for the accompanying dockerized private network. So, the `config.js.template` must be present somehow for the GitHub workflow to move to `/src` folder and use it on CI.
https://github.com/HathorNetwork/hathor-wallet-headless/blob/a8151b2e1bdbad5e7ac0d509a40172c74736a8b8/.github/workflows/integration-test.yml#L34

If the main configuration file is not a good place for these settings to be, an option would be making a new file. Here follow my suggested change: moving the specific integration test configurations to [a `test.config.js` file](https://github.com/HathorNetwork/hathor-wallet-headless/commit/02d690b45c2e77f7c660a97d708b003ad3ba6952), on the git source control.

As one of the comments here was about changing the configurations via environment variables, I also added another feature to override this config with the environment. Having two config files _and_ environment variables to allow for an application to run seems a bit confusing to me, but maybe we have to overdo it a little in order to reach a final solution.

Could @pedroferreira1 and @andreabadesso validate this suggestion?

Reply:
Me too